src/Raven.Server/Documents/ETL/EtlTransformer.cs

```diff
@@ -377,6 +379,11 @@ private JsValue HasTimeSeries(JsValue self, JsValue[] args)
             return false;
         }
         
+        private JsValue GetRevisionsCount(JsValue self, JsValue[] args) => 
+            (Current.Document.Flags & DocumentFlags.HasRevisions) == DocumentFlags.HasRevisions 
+                ? Database.DocumentsStorage.RevisionsStorage.GetRevisionsCount(Context, Current.DocumentId)
+                : JsValue.Null;
```

Comment:
I wonder if returning `null` if a document has no revisions is good. I'd rather expect the method `getRevisionsCount(this)` to return 0 in that case.

Reply:
ok